TEST_APPS/readme.md

```diff
@@ -89,5 +98,6 @@ Icetea prints the results from the test run to the command line, and the final r
 +---------------+----------------+
 ```
 
-The results from the tests can also be found in the mbed-os/log folder.
-You probably want to add the log folder to your .mbedignore file to prevent issues with build commands becoming too long over the time.
+You can also find the results from the tests in the `mbed-os/log` folder.
+
+You probably want to add the log folder to your `.mbedignore` file to prevent issues with build commands becoming too long over the time.
```

Comment:
why build command touch even log files? Could it be ignored by default?

Reply:
@SeppoTakalo @OPpuolitaival Could y'all address this question?

Reply:
I believe the concern is that `mbed-os/log` adds another include directory to the already long list of include directories. As long as no subdirectories are added to `mbed-os/log`, I think it'd be safe to remove this section in the docs.

Reply:
If we know that its always `mbed-os/log` then I would be happy to add that into `.mbedignore` file, just to be safe.